docs/conf.py

```diff
@@ -52,9 +52,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '22.10'
+version = '22.11'
 # The full version, including alpha/beta/rc tags.
-release = '22.10.0'
+release = '22.11.0-SNAPSHOT'
```

Comment:
```suggestion
release = '22.10.0'
```

Changes to this file should not be committed